target_chains/ethereum/test/pyth.js

```diff
@@ -954,14 +947,8 @@ contract("Pyth", function () {
       2
     );
 
-    // This test fails without the hard coded gas limit.
-    // Without gas limit, it fails on a random place (in wormhole sig verification) which
-    // is probably because truffle cannot estimate the gas usage correctly. So the gas is
-    // hard-coded to a high value of 6.7m gas (close to ganache limit).
     await expectRevertCustomError(
-      this.pythProxy.executeGovernanceInstruction(transferBackVaaWrong, {
-        gas: 6700000,
-      }),
```

Comment:
This took hours from me as was not working with newer ganache (caused by lock file change), but apparently without gas specification it works now unlike the past!